neural_solution/examples/hf_models/README.md

```diff
@@ -118,6 +122,19 @@ optional arguments:
 ``` shell
 [user@server tf_example1]$ curl -X GET  http://localhost:8000/download/{task_id} --output quantized_model.zip
 # download quantized_model.zip
+```
+### Manage resource
+```shell
+# query cluster information
+neural_solution cluster --query
+
+# add new node into cluster
+# parameter: "<node1> <number_of_sockets> <number_of_threads>;<node2> <number_of_sockets> <number_of_threads>"
```

Comment:
A good explanation for the node parameter format. I suggest adding it to the document as well.